components/clearbutton/stories/template.js

```diff
@@ -25,6 +27,7 @@ export const Template = ({
 				"is-disabled": isDisabled,
 				...customClasses.reduce((a, c) => ({ ...a, [c]: true }), {}),
 			})}
+			style=${ifDefined(styleMap(styles))}
 			?disabled=${isDisabled}
 		>
 			<div class="${rootClass}-fill">
```

Comment:
I think it would be just below this line, we are hard coding `iconName: "Cross100",` within the icon which is resulting in the icon not changing size when you toggle the t-shirt size in the story controls.